src/plugins/dashboard/server/saved_objects/dashboard_migrations.ts

```diff
@@ -257,14 +307,8 @@ export const createDashboardSavedObjectTypeMigrations = (
     '7.0.0': flow(migrations700),
     '7.3.0': flow(migrations730),
     '7.9.3': flow(migrateMatchAllQuery),
+    '7.10.0': flow(migrateExplicitlyHiddenTitles),
```

Comment:
Let's change this to happen in 7.17.3 to pick up all the cases of instances that are already past 7.10 who are still having the issue.

Reply:
Even though it's very unlikely that this change will actually be able to fix any cases, due to most of the in-between versions not having https://github.com/elastic/kibana/pull/95355, I think this is a good call, just in case. It can't hurt anything. 

Will update the version!